server/serverAPI/model/recipe/IRecipe.ts

```diff
@@ -43,12 +48,12 @@ export default interface IRecipe extends IBaseRecipe {
     readonly preparationInMinutes: number;
 
     /**
-     * Type of the dish that is going to be cooked. e.g. main dish, dessert etc.
+     * Approximate total cost of the meal.
```

Comment:
Are we only using USD? If so, pls specify if this is total cost in dollars, cents, etc